pycobertura/cobertura.py

```diff
@@ -47,7 +49,15 @@ def __init__(self, report, filesystem=None):
         files referenced in the report. Please check the pycobertura.filesystem
         module in order to discover more about filesystems.
         """
-        self.xml = ET.parse(report).getroot()
+        try:
+            self.xml = ET.fromstring(report).getroot()
+        except lxml.etree.XMLSyntaxError:
```

Comment:
Can we reuse the module `ET`?

```suggestion
        except ET.XMLSyntaxError:
```

Or stop using `ET` and use `lxml.etree` instead? I don't have a strong preference, but let's use one or the other for consistency.

Reply:
oh sorry about that. I have forgotten `ET`.
I would replace `lxml.etree` with `ET`.